### ISSUE TITLE
Now 2 Config Question

### ISSUE THREAD

**Issue by seejamescode** [hanford/next-offline#114]
Hey @hanford, thanks a bunch for this plugin! We are using it over at https://github.com/pingboard/next-pwa-boilerplate/ because it is so easy. I am currently updating that boilerplate to use Now 2.

I was just hoping you had a minute to explain the [next.config.js](https://github.com/hanford/next-offline/blob/master/examples/now2/next.config.js) in your Now 2 example. Is checking that the module exists a bug with Now 2 or is there a risk of the module not being found on build?

**Comment by hanford** [comment]
@seejamescode yeah, not 100% certain if that is required anymore. At one point during the deployment, Now 2.0 wouldn't find the dependency and it would fail. Super strange.. but potentially it's been fixed? I wrote the Now 2.0 within days of Now 2.0 announcement, so it was very early.


If you can confirm that's no longer needed, I'd be very happy to update the example.

I am doing it in all of my Now 2.0 apps:
https://github.com/hanford/trends/blob/master/www/next.config.js
https://github.com/hanford/personal-website/blob/master/next.config.js

**Comment by seejamescode** [comment]
Thanks for you help!
